crates/holochain_sqlite/src/sql/peer_meta_store/schema/0.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE peer_meta (
+  id INTEGER PRIMARY KEY UNIQUE NOT NULL,
+  peer_url TEXT NOT NULL,
+  meta_key TEXT NOT NULL,
+  meta_value BLOB NOT NULL,
+  expires_at INTEGER,
+  UNIQUE(peer_url, meta_key) ON CONFLICT REPLACE
+);
```

Comment:
I don't actually know if this is better, but just another option : )

```suggestion
CREATE TABLE peer_meta (
  peer_url TEXT NOT NULL,
  meta_key TEXT NOT NULL,
  meta_value BLOB NOT NULL,
  expires_at INTEGER,
  PRIMARY KEY (peer_url, meta_key) ON CONFLICT REPLACE
);
```

Reply:
With the unique constraint not being named, it's no easier to migrate than the simpler version - let's do this